src/main/java/com/flightstats/hub/dao/aws/S3WriteQueue.java

```diff
@@ -47,24 +44,30 @@
     @Inject
     private StatsdReporter statsdReporter;
 
+    S3WriteQueue(ContentDao spokeWriteContentDao, ContentDao s3SingleContentDao, StatsdReporter statsdReporter) {
+        this.spokeWriteContentDao = spokeWriteContentDao;
+        this.s3SingleContentDao = s3SingleContentDao;
+        this.statsdReporter = statsdReporter;
+    }
+
     @Inject
     private S3WriteQueue() throws InterruptedException {
-        logger.info("queue size {}", QUEUE_SIZE);
+        log.info("queue size {}", QUEUE_SIZE);
         for (int i = 0; i < THREADS; i++) {
             executorService.submit(() -> {
                 try {
                     while (true) {
                         write();
                     }
                 } catch (Exception e) {
-                    logger.warn("exited thread", e);
+                    log.warn("exited thread", e);
                     return null;
                 }
             });
         }
     }
 
-    private void write() throws InterruptedException {
+    void write() throws InterruptedException {
```

Comment:
If this is no longer private because of testing, I tend to annotate this type of thing with `@VisibleForTesting`

Reply:
Yes, I can annotate it